src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/PropertySetAnalysis/PropertySetAnalysis.PropertySetDataFlowOperationVisitor.cs

```diff
@@ -173,6 +221,63 @@ public override PropertySetAbstractValue VisitObjectCreation(IObjectCreationOper
             protected override PropertySetAbstractValue VisitAssignmentOperation(IAssignmentOperation operation, object argument)
             {
                 PropertySetAbstractValue baseValue = base.VisitAssignmentOperation(operation, argument);
+
+                if (this.TrackedFieldPropertyAssignmentsOpt != null
+                    && this.TrackedTypeSymbol.Equals(operation.Target.Type)
+                    && (operation.Target.Kind == OperationKind.PropertyReference
+                        || operation.Target.Kind == OperationKind.FieldReference
+                        || operation.Target.Kind == OperationKind.FlowCaptureReference))
+                {
+                    AnalysisEntity targetAnalysisEntity = null;
+                    if (operation.Target.Kind == OperationKind.FlowCaptureReference)
+                    {
+                        PointsToAbstractValue lValuePointsToAbstractValue = this.GetPointsToAbstractValue(operation.Target);
+                        if (lValuePointsToAbstractValue.LValueCapturedOperations.Count == 1)
+                        {
+                            IOperation lValueOperation = lValuePointsToAbstractValue.LValueCapturedOperations.First();
+                            if (lValueOperation.Kind == OperationKind.FieldReference
+                                || lValueOperation.Kind == OperationKind.PropertyReference)
+                            {
+                                this.AnalysisEntityFactory.TryCreate(lValueOperation, out targetAnalysisEntity);
+                            }
+                        }
+                        else
+                        {
+                            Debug.Fail("Can LValues FlowCaptureReferences have more than one operation?");
```

Comment:
I think this can happen for case such as `(a ?? b)?.c`

Reply:
Actually let me correct it: `(a ?? b)[0] = c`

Reply:
I added a unit test, `InsecureButNotInitialized_Instance_Constructor_Interprocedural_LValuesWithMoreThanOneCapturedOperation_NoDiagnostic`.  Didn't have more than one LValueCapturedOperations.

None of the unit tests have an example of code with that produces a PointsToAbstractValue with more than one LValueCapturedOperations.

Reply:
Hmm in that case I need to search more tests :) Though even if it is possible, IMO it is highly unlikely anyone will write such code, so I am fine with ignoring this case.

Reply:
That's my plan 😁